src/lib/utilities/static.ts

```diff
@@ -1593,12 +1603,12 @@ export const staticUtilities: StaticUtility = {
     },
     'meta': {
       'group': 'textDecoration',
-      'order': 3,
+      'order': 4,
     },
   },
 
   // http://localhost:3001/utilities/typography.html#text-decoration-style
-  'underline-solid': {
+  'decoration-solid': {
```

Comment:
I think we could support both `underline-x` and `decoration-x` at the same time to avoid breaking changes

Reply:
2964fff adds `underline-x` fallbacks

Reply:
b7cea88 adds static `underline-x` fallbacks